package/MDAnalysis/coordinates/XTC.py

```diff
@@ -26,15 +37,15 @@ class XTCWriter(XDRBaseWriter):
     they require significantly less disk space and the loss of precision is
     usually not a problem.
 
-    Parameter
-    ---------
-    filename : str
+    Parameters
+    ----------
+    filename: str
```

Comment:
I used spaces around colons without problems; for the [HOLE docs](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections) I tried to follow all the NumpDocs word-by-word in order to learn them and get me to write proper doc strings.